src/zero/jdbc.clj

```diff
@@ -0,0 +1,270 @@
+(ns zero.jdbc
+  "clojure.tools.logging wrapping for clojure.java.jdbc"
+  (:require [clojure.java.jdbc :as jdbc]
+            [clojure.tools.logging.readable :as log]))
+
+(defmacro query
+  "Logged alias for [[clojure.java.jdbc/query]]:
+
+  Given a database connection and a vector containing SQL and optional parameters,
```

Comment:
Do not copy and paste from the clojure doc. Regardless of how i feel about copying work without accreditation, this is yet another thing we have to maintain. IT WILL GO OUT OF DATE.

Reply:
The attribution aspect slipped my mind, deleted.

Reply:
I think it's fine if you do credit the original work, but I'd more likely use their summary and link to their doc for examples.